frigg/webhooks/test_commands.py

```diff
@@ -16,13 +17,25 @@ def load_fixture(self, fixture):
         fixtures_path = os.path.join(settings.BASE_DIR, self.FIXTURE_PATH, fixture)
         return json.load(open(fixtures_path, encoding='utf-8'))
 
-    def test_command(self):
-        r = redis.Redis(**settings.REDIS_SETTINGS)
-        r.lpush(settings.FRIGG_WEBHOOK_QUEUE, json.dumps({
+    def setUp(self):
+        self.maxDiff = None
```

Comment:
[Making long diff print as diffs and not something like "you have a long diff"](https://docs.python.org/3.5/library/unittest.html#unittest.TestCase.maxDiff)